.buildkite/pipeline.yml

```diff
@@ -9,11 +9,16 @@ steps:
 
   - type: "waiter"
 
-  - name: ":pill:"
+  - name: ":pill: lint"
```

Comment:
did you get these labels backwards? test is labelled as lint and lint is labelled as test